### DISCUSSION TITLE
Restore after format

### DISCUSSION THREAD

**Discussion by vidario75** [t1m0thyj/WinDynamicDesktop#495] [General]
Hi all, I am about to format my pc and would like to reinstall  WinDynamicDesktop exactly as it is now (Themes - Global Settings, etc). What should I save? Thank you!

**Comment by t1m0thyj** [comment]
Hello, sorry if this answer is too late but if you navigate to the folder where WDD is installed (see [here](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Troubleshooting#finding-where-app-is-installed) if unsure where that is), then you can copy over the "settings.json" file and the "scripts" and "themes" directories to your new installation.
